unsymlink-lib

```diff
@@ -11,9 +12,33 @@ import subprocess
 import sys
 
 
+def decode(value):
+    if isinstance(value, bytes):
+        return value.decode('utf-8', errors='replace')
```

Comment:
I think `backslashreplace` will be better here.

Reply:
Unfortunately, it causes `UnicodeDecodeError` for Python 2.7.
Please see #7.